collect_app/src/main/java/org/odk/collect/android/fragments/DataManagerList.java

```diff
@@ -149,7 +143,7 @@ protected void onResume() {
     }
 
     @Override
-    protected void onPause() {
```

Comment:
I guess they made the method public because the LifeCycle of the fragment can be controlled by the activity class as well